### PR TITLE
Uppmax config and download utils adjustments

### PR DESCRIPTION
**Pull request by mtstahl**
This PR adds
* the configuration and profile for running the pipeline on Uppmax nodes,
* definition of `cwd` in `utils/download-references.sh` (otherwise all the downloads are stored to `~`)
* adds the `-N` parameter to `wget` calls in `utils/download-references.sh` to download only non-existing or newer files.

(My first PR ever ...)

### CODE REVIEW

**Comment by ewels** [comment]
Hi @mtstahl,

Congratulations on your PR - looks great! 🎉 

Note that this pipeline already has an UPPMAX config, though it's loaded remotely so not in this config file. You can find it here instead: https://github.com/nf-core/configs/

`-profile uppmax` should already work though.

Phil

**Comment by mtstahl** [comment]
All right, thanks @ewels. Just learning a lot of stuff. :)

**Comment by ewels** [comment]
It's a brand new feature and not really documented anywhere yet, so you weren't to know..!

**Comment by mtstahl** [comment]
Just changes `utils/download-references.sh` now.

**Comment by matq007** [comment]
The `download-references.sh` takes the desired path as input so if you want to install stuff to a different path you can just run `cd utils && sh download-references.sh /new/path`

**Comment by mtstahl** [comment]
Yes, I specified a path and it installed the star references to this path. Then it was executing `cd $cwd` and that ended always up in `~` as `cwd` was not defined.

File: utils/download-references.sh [hunk 1]
```diff
@@ -5,31 +5,31 @@ if [ $# -eq 0 ]; then
 else
     DEST=$1
     if [ -d "$DEST" ]; then
-        mkdir $DEST/rnafusion_references
-        cd $DEST/rnafusion_references
-
+        cwd=$DEST/rnafusion_references
+        mkdir $cwd
```
Comment (matq007) [inline utils/download-references.sh]: Can you make it a one-liner?
`mkdir $cwd && cd $cwd`

**Comment by matq007** [comment]
Nice catch Matthias! 👍

File: utils/download-references.sh (hunk 1 above)
Comment (mtstahl) [reply utils/download-references.sh]: done

**Pull request merged**